src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 > &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

Comment:
You should have possibility to change amount of active stars by changing a modifier. You could use code from `stars` task. 

And try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)